incubator/neo4j/values.yaml

```diff
@@ -0,0 +1,73 @@
+# Default values for Neo4j.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+name: "neo4j"
+
+# Specs for the Neo4j docker image
+image: "neo4j"
+imageTag: "3.2.2-enterprise"
+imagePullPolicy: "Always"
+
+# Specs for the images used for running tests against the Helm package
+testImage: "markhneedham/k8s-kubectl"
+testImageTag: "master"
+
+# Ports
+boltPort: 7687
+httpPort: 7474
+discoveryPort: 5000
+txPort: 6000
+raftPort: 7000
+
+# Cores
+core:
+  name: core
```

Comment:
Should I hardcode those in the yaml file or is there a way to put them in the values file and not have them configurable?